init.rb

```diff
@@ -16,8 +16,13 @@
   name 'Redmine Issue Badge plugin'
   author 'Akiko Takano'
   description 'Plugin to show the number of assigned issues with badge on top menu.'
-  version '0.0.2'
+  version '0.0.3'
   url 'https://github.com/akiko-pusu/redmine_issue_badge'
   author_url 'http://twitter.com/akiko_pusu'
   requires_redmine :version_or_higher => '2.5.0'
+
+  settings :partial => 'settings/redmine_issue_badge',
+    :default => {
+      'activate_for_all_users' => 'false'
+    }
```

Comment:
__[rubocop]__
* Indent the right brace the same as the start of the line where the left brace is.